platform/server/elasticsearch/__tests__/call_api.test.ts

```diff
@@ -0,0 +1,78 @@
+import { callAPI } from '../call_api';
+
+test('should call the api when it exists, with the right context and params', async () => {
+  let apiContext;
```

Comment:
You can do `let apiContext: any` to fix the `'this' implicitly has type 'any' because ...` below

Reply:
that warning message still shows up for me!

Reply:
just to document, the fix was to specify the `this` type in the anonymous function params.
```js
let apiContext;
  const fooFn = jest.fn(function(this: any) {
    apiContext = this;
  });
```

Reply:
And here is the doc for `this` params: https://www.typescriptlang.org/docs/handbook/functions.html#this-parameters